bin/run_tests.py

```diff
@@ -42,12 +42,15 @@
         sys.executable,
         "-m",
         "pytest",
+        "--dist",
+        "loadgroup",
```

Comment:
This allows the build to run all the iOS tests on the same machine; see the iOS tests for the reason.